### ISSUE TITLE
[Unblock Legitimate Site]

### ISSUE THREAD

**Issue by AlexaBNBM** [MetaMask/eth-phishing-detect#69457]
**Domain:** bnbmixer.com / bnbmixer.eth.limo

**Details:**
*bnbmixer.com is one of the few legitimate privacy provider on the BSC, it has been here for years and is not a scam.
This tool provides a signed Letter of Guarantee for every session (before any deposit), and this letter can be checked using the Public PGP key displayed on the homepage.
Therefore, any serious accusation regarding this service should come with the associated letter of guarantee. Then it is easy to check, by looking at the wallet's addresses included in that signed letter of guarantee.
The user that reported that issue joined GitHub 3days ago only to report this legitimate service without the simple and incontestable evidence (the letter).

* That person who reported is most likely a scammer who is about to create a copy of that legitimate website and want the genuine one to be block in order to lure users into his own scam... (We've seen that again and again... Every time a legitimate service is down, scammers spoof them and scam users...)

Therefore, I am asking for you to unblock that Legitimate Site.*

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexaBNBM** [comment]
Here is the way to check it:
I just crated an order: https://www.bnbmixer.com/mixing.php?id=6739982654
You can download the letter of guarantee on that page, here is the content of that letter:

> -----BEGIN PGP SIGNED MESSAGE-----
> Hash: SHA1
> 
> Mixing ID: 6739982654
> Mixing Wallet: 0x1ff51adc370bCd9216248bF56937a0415d73a449
> Forwarding Wallet:
> 	- 0x0123456789012345678901234567890123456789
> Minimum deposit limit: 0.1 BNB.
> 
> In order to provide its mixing services,
> www.bnbmixer.com hereby generated the above Mixing Wallet.
> For every BNB deposited into this Mixing Wallet within the allowed limits above-mentioned,
> www.bnbmixer.com will forward the same amount of BNB, minus fees, to the above Forwarding Wallet(s).
> Thank you for using our service!
> -----BEGIN PGP SIGNATURE-----
> Version: GnuPG v2.0.22 (GNU/Linux)
> 
> iQGcBAEBAgAGBQJmlhQvAAoJEJWMxo6pBtXq9qYL/3ooJqBRsbE8z+IP1euq+qkn
> bNha3HhhewvjiuW90HO8cuylKyCGD4nFautqdtFBX0OYLzHpahHYlP9gmw8CUo+Y
> rFtc9P+yVR9vF0EZjVZPNW/E/rkdUjnOSxIyqkwFmfPFIVu1pZ1idmDVldLF4LbO
> KE5COT4/m86QCUDUtu99HlG7wkGaLkLpOijVvyWptK0OppUhOkto6srri8S+WsuQ
> 31wxafb6TAv/bh7jZ/sOGdVUKK63Cit8G2wIeI6cWeNr1cN5cAN3mqxIwAd+poPR
> 4uo3n3A/14gnsF11PyBm5RDCXM7GphJJQ9gX1PW1PnihXEvkd5o8+pxUTjpiLsRQ
> 0lUHlwomlBZWSCpyNlAz1N8CxMt/WpP7n0c06V1m3xl4SSvhdJrcmB4KAnMOZ/3a
> qyQatBltVQtZZNb5KhwxmcvlZ871WwzJbVqvAIaZHeFgMo4M0GZW6L/FxewwkptB
> ZiW4IbkFDlQWnxTo0OZG7TDMuG0dFYPpoLCEvVGURg==
> =CIXH
> -----END PGP SIGNATURE-----

You can verify that the letter is authentic by using tools like: https://pgptool.org
Click on the tab **verify** paste that letter in **PGP-signed Message** and paste the public key from bnbmixer's home page in **Signer's Public Key**
You will get: **Message signature is verified with fingerprint: 19be5ab05057b6bad3ecf895958cc68ea906d5ea**
That will confirm the Letter is Authentic, and now to check, you simply go check the **Mixing Wallet** (using bscscan) and see if any deposit has been made. If yes, you then go on the **Forwarding Wallet** if there is no transfer (after the forwarding delay specified in the order), then you can conclude the service was not done...

It would be the only and ultimate proof of any wrongdoing, because that letter can't be tampered with as it should be downloaded by the user before any deposit.

**Comment by AlexaBNBM** [comment]
On 07/13, user *Sentolone* reported bnbmixer: https://github.com/MetaMask/eth-phishing-detect/issues/69264
and **on that exact same day**: a phishing domain has been bought **bnb-mixers.xyz** and now host a scam: https://who.is/whois/bnb-mixers.xyz (07-13)

That's exactly what I thought... please take action asap.

**Comment by AlexHerman1** [comment]
@AlexaBNBM  you just signed up for github 7 hours ago, and youre criticizing a reporter for the same thing?

**Comment by AlexaBNBM** [comment]
Dear @AlexHerman1
Sorry for the misunderstanding, I might have express myself in a wrong way.
I am not criticizing anyone because "new", I only pointed the date of his registration because it matched the date of someone buying a similar domain name (which host a scam).
And it seemed to confirm my intuition that I expressed before "Why Would someone make a fake report about a legit website ? probably because he tried to get it down and scam users with a similar domain..."

So my message was just to highlight the fact that someone making fake report about a legit service which provides its users with a signed letter that can easily be checked to prove any wrongdoing (but because this report was fake, that user didn't come with a letter of guarantee: Letter of guarantee can't lie).

I personally joined Github especially to contest that ban, as BNB Mixer is a legit service that has been around for many years.
However if that report came with a signed letter, then I would have no option than to agree regarding the wrongdoing (again, letter of guarantee doesn't lie), but that is not the case.

Thanks for your time,
hoping I expressed my POV in a better way this time.
